### PR TITLE
images to assets

### PR DESCRIPTION
**Pull request by bcdunbar**
adresses https://github.com/plotly/dash-docs/issues/144#issuecomment-414659124 in conjunction with https://github.com/plotly/dash-docs/pull/164

- [x] mv images to assets
- [x] deploy and check on DDS

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -4,11 +4,11 @@
 import config
 
 def StaticUrlPath(resource):
-    if not os.path.exists(os.path.join('static', resource)):
+    if not os.path.exists(os.path.join('assets/images', resource)):
```
Comment (chriddyp) [inline utils.py]: Use `os.path.join('assets', 'images', resource)` for cross platform file paths

**Comment by T4rk1n** [comment]
I think we can get rid of `utils.StaticUrlPath`, it is not used anymore. There is also `app.get_asset_url`  to retrieve the local assets url, it is automatically prefixed with `requests_pathname_prefix`.

**Comment by chriddyp** [comment]
We use `utils.StaticUrlPath` as an example in the `Header` function still: https://github.com/plotly/dash-on-premise-sample-app/blob/master/components/Header.py. Not sure if we can import `app` there or not (i.e. will there be recursive imports with `import app` in `components/Header.py` and `import components` in `app.py`?)

**Comment by T4rk1n** [comment]
> will there be recursive imports with import app in components/Header.py and import components in app.py

It will be a circular import unless app is defined before `import components` which I don't recommend since it's against PEP.

One pattern I use to counter this is to make 'factories' functions that takes in an app and return a component.

**Comment by bcdunbar** [comment]
@chriddyp @T4rk1n I've updated the path. What was the conclusion from the above? Should I merge as is or rework and utilize  `get_asset_url`?

**Comment by cldougl** [comment]
@michaelbabyn can you resolve the conflicts/ update this pr

**Comment by michaelbabyn** [comment]
@cldougl , This is good to merge. I thought that it would be better to use the app.get_asset_url pattern since that's a method I use/recommend pretty often.

**Comment by cldougl** [comment]
@michaelbabyn perfect! 💃

**Pull request merged**